test/test_autograd.py

```diff
@@ -3127,6 +3127,38 @@ def test_set_data_preserve_pyobj(self):
         b.data = a
         self.assertTrue(b_id_saved == id(b))
 
+    def _run_and_capture_stderr(self, code):
+        import os
+        import subprocess
+
+        env = os.environ.copy()
+        env["PYTORCH_API_USAGE_STDERR"] = "1"
+        pipes = subprocess.Popen(
+            [sys.executable, '-c', code],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            env=env)
+        return pipes.communicate()[1].decode('ascii')
+
+    @unittest.skipIf(IS_WINDOWS, "Skip for windows")
```

Comment:
Prefer to mention exactly why you are skipping on Windows (in this case, it doesn't work. Might be worth filing a bug for it and mentioning it here.)